src/views/v2/emails/template-parts/header/head/json-ld.php

```diff
@@ -28,4 +28,7 @@
 	return;
 }
 
-// @todo @juanfra @codingmusician @rafsuntaskin print JSON-LD here.
+?>
+<script type="application/ld+json">
+	<?php echo wp_json_encode( $json_ld ); ?>
```

Comment:
Pass a second param `$json_options` that is filtered.
https://github.com/the-events-calendar/the-events-calendar/blob/master/src/views/v2/components/data.php#L52-L62